sol6-fallback-receive-split.md

```diff
@@ -0,0 +1,17 @@
+# Solidity 0.6.x features: Fallback functions split
+
+In versions of Solidity before 0.6.x, developers typically added a [fallback function](https://solidity.readthedocs.io/en/v0.5.15/contracts.html#fallback-function) as a function that a contract executed if no other functions matched the function called, or if a contract received ether, but no data.
```

Comment:
Please keep the lines short to help commenting :(

Reply:
Sorry writing in short lines is not very writer-friendly, and especially when I am in blog writing mode I forget to line break. Will fix…